plugins/CoreHome/angularjs/reporting-menu/reportingmenu-model.js

```diff
@@ -161,5 +163,38 @@
                 return model.menu;
             });
         }
+
+        function fetchVisits(date, period, callback) {
+            var ajaxRequest = new ajaxHelper();
+
+            ajaxRequest.addParams({
```

Comment:
As this is an angular component you should use `piwikApi` in favor of the ajaxHelper I guess. See https://github.com/matomo-org/matomo/blob/4.x-dev/plugins/PrivacyManager/angularjs/manage-gdpr/managegdpr.controller.js#L24  for an example

Reply:
I was wondering why we have both, but I see now 😄 
thanks